dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/StringUtils.java

```diff
@@ -38,144 +38,6 @@ private StringUtils() {
-        return str == null ? null : str.trim();
-    }
-
-    /**
-     * <p>Returns either the passed in CharSequence, or if the CharSequence is
-     * whitespace, empty ("") or {@code null}, the value of {@code defaultStr}.</p>
-     *
-     * @param <T> the specific kind of CharSequence
-     * @param str the CharSequence to check, may be null
-     * @param defaultStr  the default CharSequence to return
-     *  if the input is whitespace, empty ("") or {@code null}, may be null
-     * @return the passed in CharSequence, or the default
-     */
-    public static <T extends CharSequence> T defaultIfBlank(final T str, final T defaultStr) {
-        return isBlank(str) ? defaultStr : str;
-    }
-
-    /**
-     * <p>Compares two String, returning {@code true} if they represent
-     * equal string, ignoring case.</p>
-     *
-     * @param str1  the first String, may be null
-     * @param str2  the second String, may be null
-     * @return {@code true} if the String are equal, case insensitive, or
-     *  both {@code null}
-     */
-    public static boolean equalsIgnoreCase(String str1, String str2) {
-        return str1 == null ? str2 == null : str1.equalsIgnoreCase(str2);
-    }
-
-    /**
-     * <p>Joins the elements of the provided Collection into a single String
-     * containing the provided Collection of elements.</p>
-     *
-     * @param collection the collection, may be null
-     * @param separator the separator
-     * @return a single String
-     */
-    public static String join(Collection<?> collection, String separator) {
-        return collection == null ? null : join(collection.iterator(), separator);
-    }
-
-    /**
-     * <p>Joins the elements of the provided Iterator into a single String
-     * containing the provided Iterator of elements.</p>
-     *
-     * @param iterator the iterator, may be null
-     * @param separator the separator
-     * @return a single String
-     */
-    public static String join(Iterator<?> iterator, String separator) {
-        if (iterator == null) {
-            return null;
-        } else if (!iterator.hasNext()) {
-            return "";
-        } else {
-            Object first = iterator.next();
-            if (!iterator.hasNext()) {
-                return first == null ? "" : first.toString();
-            } else {
-                StringBuilder buf = new StringBuilder(256);
-                if (first != null) {
-                    buf.append(first);
-                }
-
-                while (iterator.hasNext()) {
-                    if (separator != null) {
-                        buf.append(separator);
-                    }
-
-                    Object obj = iterator.next();
-                    if (obj != null) {
-                        buf.append(obj);
-                    }
-                }
-                return buf.toString();
-            }
-        }
-    }
-
     public static String escapeJava(String str) {
```

Comment:
You can find these methods are also copied from third party libs.

https://commons.apache.org/proper/commons-lang/javadocs/api-2.6/org/apache/commons/lang/StringEscapeUtils.html

Reply:
Thanks, and then we can remove the StringUtils now.